src/environments/environment.prod.ts

```diff
@@ -1,3 +1,6 @@
 export const environment = {
     production: true,
+    langKey: 'lang',
+    defaultLang: 'en',
+    api_url: 'http://localhost:3000'
```

Comment:
we shouldn't hardcode localhost as our route
what will be when we will deploy our app to some domain?

Reply:
I see, but it's just good to change in one place, isn't it? Or maybe there is another way to do it?